readthedocs/doc_builder/environments.py

```diff
@@ -819,7 +813,7 @@ def create_container(self):
                 host_config=self.get_container_host_config(),
                 detach=True,
                 user=settings.RTD_DOCKER_USER,
-                runtime=docker_runtime,
+                runtime="runsc",  # gVisor runtime
```

Comment:
I wonder if this should be an application setting, we don't strictly need gVisor for local development. Otherwise, we should update our install docs or something to mention the additional system dependency on gVisor. Having parity with prod is always nice though.

Reply:
We have this guide https://dev.readthedocs.io/en/latest/guides/gvisor.html

Maybe we need to merge it / mention it from the development installation instructions.

I prefer if we use gVisor in development, for parity as well. I've been using it and I had no problems.

Reply:
Yeah, linking to that in the install procedure is enough. I'm pretty sure we all went through this install already even, so it won't be a breaking change to core team.